charts/index.yaml

```diff
@@ -1,72 +1,6 @@
 apiVersion: v1
 entries:
   meshery:
-  - apiVersion: v2
```

Comment:
These changes should not be included in this PR. Perhaps, you'll need to resync your fork, @shahkushal38?